src/main/java/com/contrast/labs/ai/mcp/contrast/AssessService.java

```diff
@@ -27,9 +27,6 @@
 import com.contrast.labs.ai.mcp.contrast.sdkexstension.data.application.Application;
 import com.contrast.labs.ai.mcp.contrast.sdkexstension.data.sca.LibraryObservation;
 import com.contrast.labs.ai.mcp.contrast.sdkexstension.data.sessionmetadata.SessionMetadataResponse;
-import com.contrast.labs.ai.mcp.contrast.sdkexstension.data.traces.MetadataItem;
-import com.contrast.labs.ai.mcp.contrast.sdkexstension.data.traces.SessionMetadata;
-import com.contrast.labs.ai.mcp.contrast.sdkexstension.data.traces.TraceExtended;
 import com.contrastsecurity.models.*;
```

Comment:
importing `*`  from a package is generally not something we do. We recommend only importing needed classes from a package.

Reply:
There are also several classes that are not being used but still imported